api.ts

```diff
@@ -84,6 +85,51 @@ export class Request extends ServerRequest implements APIRequest {
         await this.send(JSON.stringify(data, replacer, space), 'application/json; charset=utf-8')
     }
 
+    async decodeBody(type: "text"): Promise<string>
+    async decodeBody(type: "json"): Promise<any>
+    async decodeBody(type: "form-data"): Promise<FormDataBody> {
```

Comment:
here should be:
```typescript
    decodeBody(type: 'text'): Promise<string>
    decodeBody(type: 'json'): Promise<any>
    decodeBody(type: 'form-data'): Promise< FormDataBody >
    async decodeBody(type: string): Promise<any> {
       if (type === "text") {
          ...
       }
    }
```

Reply:
Ok, now the Deno compiler and the VSCode intellisense aren't angry with me :smile: